src/ucs/sys/sock.c

```diff
@@ -426,7 +426,10 @@ ucs_socket_handle_io_error(int fd, const char *name, ssize_t io_retval, int io_e
         }
     }
 
-    ucs_error("%s(fd=%d) failed: %s", name, fd, strerror(io_errno));
+    if (status != UCS_ERR_NOT_CONNECTED) { /* Don't print the error in case of
```

Comment:
why is UCS_ERR_NOT_CONNECTED different from other errors if you invoke `err_cb` for it as well?